backport-Guide.md

```diff
@@ -0,0 +1,84 @@
+# Kata Containers stable backport workflow
+
+* [Backport Workflow](#backport-workflow)
+
+This document provides a short guide on how to perform a stable backport of a
+PR or patchset in the Kata Containers repositories.
+
+It does not cover all eventualities that may be encountered whilst performing
```

Comment:
Lines 8-10 suggested rewrite: 
It does not cover all eventualities that might be encountered while performing a backport, and only documents one potential git-based workflow. It does serve as a short introduction and reminder of the basic stable backport process.